square.go

```diff
@@ -1 +1,38 @@
 package main
+
+import (
+	"fmt"
+	"math"
+)
+
+type Point struct {
+	x, y int
+}
+
+type Square struct {
+	start Point
+	a     uint
+}
+
+func (s Square) End() Point {
+	/* some hardcoded stuff
+	i used the minus sign in calculation of y axis because in the attached screenshot start.y lower then end.y (???)
```

Comment:
Pay attention to minus sign. I made the right decision?
![image](https://user-images.githubusercontent.com/31538388/68984514-967def00-0821-11ea-935e-4ae590cfd125.png)

Reply:
Yep, end.y - start.y - a